src/billing/components/PaymentInfo/PaymentForm.tsx

```diff
@@ -29,9 +30,9 @@ const PaymentForm: FC<Props> = ({errorMessage}) => {
    * a hosted iframe to render a credit card form to the UI
    */
   // TODO(ariel): uncomment this when we get to define Z
```

Comment:
let's remove that comment

Reply:
I think we can leave or change it. We need to render zuora payment form and at this point, Z may or may not have been initialized.

Reply:
Not sure i follow. We can remove that TODO comment

Reply:
Will remove. I meant because script loading code isn't in here. It's in `ZuoraPaymentForm`. So, we'd have to move that code to a central component which only handles loading ZuoraForm and doesn't use hard-coded CheckoutContext.